File: Java/백준/Bronze/30319. Advance to Taoyuan Regional/Advance to Taoyuan Regional.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String[] a = in.nextLine().split("-");
        int b = Integer.parseInt(a[1]);
        int c = Integer.parseInt(a[2]);
        if (b<9 || (b==9 && c<=16)){
            System.out.println("GOOD");
        }
        else{
            System.out.println("TOO LATE");
        }
    }
}

File: C#/백준/Bronze/30319. Advance to Taoyuan Regional/Advance to Taoyuan Regional.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split('-'),int.Parse);
            if (aa[1]<9 || (aa[1]==9 && aa[2]<=16)){
                Console.WriteLine("GOOD");
            }
            else{
                Console.WriteLine("TOO LATE");
            }
        }
    }
}